xwiki-platform-core/xwiki-platform-mail/xwiki-platform-mail-ui/src/main/resources/Mail/Translations.xml

```diff
@@ -78,7 +78,7 @@ Mail.SendMailStatus_livetable.error=Error
 Mail.SendMailStatus_livetable.actions=Actions
 Mail.SendMailStatus_livetable.resend=Resend
 mailSender.admin.resendSuccess=The mail has been resent successfully
-mailSender.admin.resendFailure=The mail has failed to be resent. Click for more details.</content>
+mailSender.admin.resendFailure=Failure resending mail. Click for more details.</content>
```

Comment:
It's usually `Failure in resending`, `Failure while resending`, etc. A simpler version would be `Failed to resend`. Either keep it consistent with the success message above (i.e. revert), or use the simpler version. We should try to keep the messages as clear and concise as possible.

Reply:
Done.